src/System.CommandLine/IOption.cs

```diff
@@ -5,12 +5,24 @@
 
 namespace System.CommandLine
 {
+    /// <summary>
+    /// Defines a symbol with an argument. 
```

Comment:
We can align this to the terminology here: https://github.com/dotnet/command-line-api/blob/main/docs/Syntax-Concepts-and-Parser.md#arguments-and-arity.